src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+
+public class BuddyBalanceSubsystem extends SubsystemBase { // TODO: Find a permanent name for buddy balance subsystem - BBalance is ambiguous
```

Comment:
this subsystem needs 
1. TunableNumbers for PID.
2. DataLog for logging the position over time
3. Shuffleboard for showing position to operator